README.md

```diff
@@ -1 +1,64 @@
-# particula
```

Comment:
Sorry my comments were meant to be in the review here, but went as comments on the commits... My bad, sloppy!

Reply:
lol, I was like maybe I shouldn't read this until you finish your edits